Add inline documentation (#90)
nethesis/nethvoice-report: ui/src/doc-inline/en/data_lost_call_1_table_lostCall.md

# By Unmanaged Calls

> Data split *by Unhandled Calls* in selected queues
> in the set period, the data is grouped according to the choice
> made in the filters.

Reviewer: ```suggestion
Data split *by Unhandled Calls* in selected queues
during the configured perido. Dare are grouped according to the chosen filter criteria.
```

Data *by Unhandled Calls* shows each caller with whom
at the end of the working day the last interaction was
negative, that is, there has been a failure or incoming not responding
to the call or outgoing having no answer to the call made

> by an agent.

Reviewer: ??
